lib/driver.js

```diff
@@ -100,6 +100,7 @@ class XCUITestDriver extends BaseDriver {
       let { device, udid, realDevice } = await this.determineDevice();
       log.info(`Determining device to run tests on: udid: '${udid}', real device: ${realDevice}`);
       this.device = device;
+      this.opts.device = device;
       this.opts.udid = udid;
       this.opts.isRealDevice = realDevice;
```

Comment:
Let's put it in `opts`. We still should have a function `isRealDevice()` which checks that value.